src/core/lib.c

```diff
@@ -5,7 +5,7 @@
 #include <SDL.h>
 #include <SDL2_gfxPrimitives.h>
 
-define TYPE_FILLED_CIRCLE 2
```

Comment:
Wow, another character that got deleted by accident, I need to figure out which keybinding I'm hitting that does it...